server/migrations/20210722181933_unpadded_ext_ids.js

```diff
@@ -0,0 +1,17 @@
+exports.up = async function (knex) {
+  const zeroPaddedRows = await knex("external_ids")
+    .select("provider_location_id", "system", "value")
+    .where("system", "cvs")
+    .where("value", "~", String.raw`^0\d+$`);
+
+  const unpaddedRows = zeroPaddedRows.map((r) => ({
+    ...r,
+    value: r.value.replace(/^0+/, ""),
+  }));
+
+  return knex.batchInsert("external_ids", unpaddedRows);
```

Comment:
Just realized this migration can fail if the database already has some existing rows like the ones we’re creating here. It looks like `.onConflict` doesn’t work with `batchInsert`, so we probably need another way to address this, either by looping and inserting or querying for unpadded values that already exist and removing them from the list before inserting.